src/jit/morph.cpp

```diff
@@ -2066,12 +2066,12 @@ GenTreePtr    Compiler::fgMakeTmpArgNode(unsigned tmpVarNum
 #if FEATURE_MULTIREG_ARGS
 #ifdef _TARGET_ARM64_
```

Comment:
I will look into removing this, but I would have to test to make sure that the assert(varTypeIsStruct(type)); doesn't fire on Unix